backend/core/src/main/java/org/sonarsource/sonarlint/core/analysis/AnalysisStartedEvent.java

```diff
@@ -62,4 +64,21 @@ public Set<URI> getFileUris() {
   public boolean isTrackingEnabled() {
     return enableTracking;
   }
+
+  public UnaryOperator<String> getFileContentProvider() {
+    return path -> files.stream()
```

Comment:
Do we need to take into account that the  filePath we set via the [RepositoryBlameCommand.seFilePath() will always have the UNIX format](https://github.com/SonarSource/sonarlint-core/blob/8dc549e82f5674928a39f2ea806e4e693a685f87/backend/commons/src/main/java/org/sonarsource/sonarlint/core/commons/GitBlameUtils.java#L79)? Do any of the paths here also need to be in UNIX format?

Reply:
That's a very good catch! Yes, we should definitely take this into account. Thanks.

Reply:
I modified the `it_should_blame_with_given_contents` test to contain inner path logix to be able to test the forward-back slash scenario.